src/test/java/mate/academy/security/AuthenticationServiceImplTest.java

```diff
@@ -18,32 +18,31 @@
 class AuthenticationServiceImplTest {
     private static final String EMAIL = "bob@i.ua";
     private static final String PASSWORD = "12345";
-    private static final Role ROLE_USER = new Role(Role.RoleName.USER);
     private User user;
     private Optional<User> optionalUser;
-    private UserService userService;
-    private RoleService roleService;
     private PasswordEncoder passwordEncoder;
```

Comment:
It can be local too.
You can set up a mock for a correct user and for a  wrong user.
But it`s up to you;